.gitignore

```diff
@@ -51,3 +51,7 @@ bazel-*
 
 # VSCode
 .vscode
+
```

Comment:
Hint: You will need less people to approve your PR if you do not modify files in the root directory =)